src/Common/src/Interop/OSX/libc/Interop.sockaddr.cs

```diff
@@ -43,7 +43,7 @@ public struct sockaddr_in
             public sa_family_t sin_family; // Address family: AF_INET
             public in_port_t sin_port;     // Port in network byte order
             public in_addr sin_addr;       // Internet address
-            private ulong padding;         // 8 bytes of padding
+            private ulong _padding;        // 8 bytes of padding
```

Comment:
This should not be changed: as per the interop guidelines, field names for native structs should be identical to the names in the native headers.